safe/impact_functions/impact_function_metadata.py

```diff
@@ -147,8 +147,9 @@ def allowed_subcategories(cls, category=None):
                 .allowed_subcategories('hazard')
         else:
             metadata_dict = cls.as_dict()
-            categories = metadata_dict['categories']
-            result = add_to_list(result, categories[category]['subcategories'])
+            categories = metadata_dict['layer_requirements']
+            result = add_to_list(result,
+                                 categories[category]['%s_types' % category])
             return result
```

Comment:
@ismailsunni: Right, it seems we don't use it anymore. Actually I don't remember why did I update it - probably it was still in use at that time.